charts/bar.go

```diff
@@ -21,7 +21,10 @@ func NewBar() *Bar {
 	c := &Bar{}
 	c.initBaseConfiguration()
 	c.Renderer = render.NewChartRender(c, c.Validate)
-	c.HasXYAxis = true
+	c.addJSOMConfig(
+		c.jsonConfig("xAxis", c.XAxisList),
+		c.jsonConfig("yAxis", c.YAxisList),
+	)
```

Comment:
Add `config func` at the specific chart.
And maybe we can make this  common to rectangular coordinate charts.